internals/utils/src/index.ts

```diff
@@ -0,0 +1,35 @@
+export {default as ARG_COMMON} from './arg-common';
+export {default as getArgs} from './get-args';
+export * from './build-state';
+export {default as sleep} from './sleep';
+export {default as createPollingFn} from './create-polling-fn';
+export * from './emoji';
+export * from './errors';
+export * from './now-error';
+export {default as exit} from './exit';
+export * from './format-date';
+export {default as formatDate} from './format-date';
```

Comment:
some files actually exported named and default exports so have to use this double export syntax